ToDos/ToDoComposerViewController.swift

```diff
@@ -46,10 +47,10 @@ class ToDoComposerViewController: UIViewController {
         titleInput.frame = CGRect(x: 20, y: view.bounds.height - 300, width: view.bounds.width - 40, height: 40)
         titleInput.backgroundColor = .white
         view.addSubview(titleInput)
+        
```

Comment:
NewToDo cancelAdd button and title don't work anymore (ToDoComposerViewController, i.e. this file,  line 35)

Reply:
so rn you're only adding the `cancel` button if `navigationController` is `nil`. you can remove this logic (ln 35) and then you'll need to update the func `cancelAdd` to pop the VC appropriately